R/dfm.R

```diff
@@ -200,17 +199,39 @@ dfm.tokenizedTexts <- function(x,
                                groups = NULL, 
                                verbose = quanteda_options("verbose"), 
                                ...) {
+    dfm(as.tokens(x),  
+        tolower = tolower, 
+        stem = stem, 
+        select = select, remove = remove, 
+        dictionary = dictionary, thesaurus = thesaurus, valuetype = valuetype, 
+        groups = groups, 
+        verbose = verbose, ...)
+}
+    
+#' @noRd
+#' @importFrom utils glob2rx
+#' @export
+dfm.tokens <- function(x, 
+                       tolower = TRUE,
+                       stem = FALSE, 
+                       select = NULL,
+                       remove = NULL,
+                       dictionary = NULL,
+                       thesaurus = NULL,
+                       valuetype = c("glob", "regex", "fixed"), 
+                       groups = NULL, 
+                       verbose = quanteda_options("verbose"), 
+                       ...) {
 
     valuetype <- match.arg(valuetype)
     # set document names if none
     if (is.null(names(x))) {
         names(x) <- paste0(quanteda_options("base_docname"), seq_along(x))
     } 
-
-    if (who_called_me_first(sys.calls(), "dfm") %in% c("tokens", "tokenizedTexts")) {
```

Comment:
This was designed to print a unique message identifying the input object from the call stack. It's far from perfect, but the old message did prevent this:
```r
> dfm(tokenize(data_char_sampletext), verbose = TRUE)
Creating a dfm from a tokenizedTexts input...
Creating a dfm from a tokens input...
   ... lowercasing
   ... found 1 document, 241 features
   ... created a 1 x 241 sparse dfm
   ... complete. 
Elapsed time: 0.018 seconds.
Document-feature matrix of: 1 document, 241 features (0% sparse).
```
Can you fix? (even though `tokenizedTexts` will soon be removed...

Reply:
Done